README.rst

```diff
@@ -13,6 +13,10 @@
 
 .. start
 
+.. Note::
+
+   This is the pybind11 **smart_holder** branch. Please refer to
```

Comment:
This won't be true when merged, right?

Reply:
Correct, that's what I want to clean out last minute, "TODO this PR" in the PR description.